packages/swing-store/src/types-index.d.ts

```diff
@@ -3,7 +3,6 @@ export type {
   SwingStoreKernelStorage,
   SwingStoreHostStorage,
 } from './swingStore.js';
-export type { KVStore } from './kvStore.js';
```

Comment:
Let's have `@agoric/swing-store` re-export the `KVStore` type.

```suggestion
export type { KVStore } from '@agoric/internal/src/kv-store.js';
```